src/audio/module_adapter/CMakeLists.txt

```diff
@@ -77,10 +77,14 @@ if((NOT CONFIG_LIBRARY) OR CONFIG_LIBRARY_STATIC)
 	endif()
 
 	if(CONFIG_DTS_CODEC)
-	add_local_sources(sof module/dts.c)
+	add_local_sources(sof module/dts/dts.c)
```

Comment:
any plan to enable DTS_CODEC or effect in sof?

Reply:
It already is, it's only available via license hence this missing code

Reply:
ok, I just don't know how to play with it, hope have a chance to have a try in future, :)